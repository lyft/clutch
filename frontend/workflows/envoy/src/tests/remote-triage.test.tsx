import React from "react";
import { shallow } from "enzyme";

import RemoteTriage from "../remote-triage";

describe("Remote Triage workflow", () => {
  let component;

  beforeAll(() => {
    component = shallow(<RemoteTriage />);
  });

  it("renders correctly", () => {
    expect(component.debug()).toMatchSnapshot();
  });
});
