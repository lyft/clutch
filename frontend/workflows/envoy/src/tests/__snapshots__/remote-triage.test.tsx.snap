// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Remote Triage workflow renders correctly 1`] = `
"<Wizard dataLayout={{...}} heading={[undefined]} maxWidth={false}>
  <TriageIdentifier name=\\"Lookup\\" />
  <TriageDetails name=\\"Details\\" />
</Wizard>"
`;
